app/src/main/java/com/infomaniak/mail/ui/MainActivity.kt

```diff
@@ -118,16 +118,17 @@ class MainActivity : ThemedActivity() {
     }
 
     private fun observeNetworkStatus() {
+        var previousIsAvailable: Boolean? = null
```

Comment:
His trigger several times?

Reply:
Exactly, it triggers twice when the network is available.

Reply:
So this is not the place to manage it

Reply:
It is necessary to do it from the `livedata` itself, so it notifies only when it is necessary

Reply:
At first, I thought the `LiveDataNetworkStatus` was bugged, but in fact it's not.
It broadcasts the global network status when any network's connectivity status changes.

So I just added a `distinctUntilChanged()` to only listen to the global network status changes.